terratest/test/k8gb_basic_roundrobin_test.go

```diff
@@ -0,0 +1,130 @@
+	k8s.CreateNamespace(t, optionsContext2, namespaceName)
+	defer k8s.DeleteNamespace(t, optionsContext1, namespaceName)
+	defer k8s.DeleteNamespace(t, optionsContext2, namespaceName)
+
+	createGslbWithHealthyApp(t, optionsContext1, kubeResourcePath, gslbName, host)
+
+	createGslbWithHealthyApp(t, optionsContext2, kubeResourcePath, gslbName, host)
+
+	ingressIPs1 := GetIngressIPs(t, optionsContext1, gslbName)
+	ingressIPs2 := GetIngressIPs(t, optionsContext2, gslbName)
+	var expectedIPs []string
+	expectedIPs = append(expectedIPs, ingressIPs2...)
+	expectedIPs = append(expectedIPs, ingressIPs1...)
+
+	sort.Strings(expectedIPs)
+
+	t.Run("round-robin on two concurrent clusters with podinfo running", func(t *testing.T) {
+		resolvedIPs5053, err := waitForLocalGSLB(t, host, 5053, expectedIPs)
+		require.NoError(t, err)
+		resolvedIPs5054, err := waitForLocalGSLB(t, host, 5054, expectedIPs)
+		require.NoError(t, err)
+
+		assert.NotEmpty(t, resolvedIPs5053)
+		assert.NotEmpty(t, resolvedIPs5054)
+		assert.Equal(t, len(resolvedIPs5053), len(resolvedIPs5053))
+		assert.ElementsMatch(t, resolvedIPs5053, expectedIPs, "%s:%s", host, 5053)
+		assert.ElementsMatch(t, resolvedIPs5054, expectedIPs, "%s:%s", host, 5054)
+	})
+
+	t.Run("kill podinfo on the first cluster", func(t *testing.T) {
+		// kill app in the first cluster
+		k8s.RunKubectl(t, optionsContext1, "scale", "deploy", "frontend-podinfo", "--replicas=0")
+
+		assertGslbStatus(t, optionsContext1, gslbName, host+":Unhealthy")
+
+		resolvedIPs5053, err := waitForLocalGSLB(t, host, 5053, ingressIPs2)
+		require.NoError(t, err)
+		resolvedIPs5054, err := waitForLocalGSLB(t, host, 5054, ingressIPs2)
+		require.NoError(t, err)
+		assert.ElementsMatch(t, resolvedIPs5053, resolvedIPs5054)
+	})
+
+	t.Run("kill podinfo on the second cluster", func(t *testing.T) {
+		// kill app in the second cluster
+		k8s.RunKubectl(t, optionsContext2, "scale", "deploy", "frontend-podinfo", "--replicas=0")
+
+		assertGslbStatus(t, optionsContext2, gslbName, host+":Unhealthy")
+
+		_, err = waitForLocalGSLB(t, host, 5053, []string{""})
+		require.NoError(t, err)
+		_, err = waitForLocalGSLB(t, host, 5054, []string{""})
+		require.NoError(t, err)
+	})
+
+	t.Run("start podinfo on the both clusters", func(t *testing.T) {
+		// start app in the both clusters
+		k8s.RunKubectl(t, optionsContext1, "scale", "deploy", "frontend-podinfo", "--replicas=1")
+		k8s.RunKubectl(t, optionsContext2, "scale", "deploy", "frontend-podinfo", "--replicas=1")
+
+		assertGslbStatus(t, optionsContext1, gslbName, host+":Healthy")
+		assertGslbStatus(t, optionsContext2, gslbName, host+":Healthy")
+
+		resolvedIPs5053, err := waitForLocalGSLB(t, host, 5053, expectedIPs)
+		require.NoError(t, err)
+		resolvedIPs5054, err := waitForLocalGSLB(t, host, 5054, expectedIPs)
+		require.NoError(t, err)
+
+		assert.NotEmpty(t, resolvedIPs5053)
+		assert.NotEmpty(t, resolvedIPs5054)
+		assert.Equal(t, len(resolvedIPs5053), len(resolvedIPs5053))
```

Comment:
```suggestion
		assert.Equal(t, len(resolvedIPs5053), len(expectedIPs))
```
or
```suggestion
		assert.Equal(t, len(resolvedIPs5053), len(resolvedIPs5054))
```
not sure, what was the original idea

Reply:
@ytsarev , thx for findings! fixed both occurrences

```go
		assert.Equal(t, len(resolvedIPs5053), len(expectedIPs))
		assert.Equal(t, len(resolvedIPs5054), len(expectedIPs))
```